include/gatekeeper_main.h

```diff
@@ -37,6 +37,12 @@
  */
 extern int gatekeeper_logtype;
 
+#define BLOCK_LOGTYPE RTE_LOGTYPE_USER1
+
+#define B_LOG(level, ...)		        \
+	rte_log_ratelimit(RTE_LOG_ ## level,	\
+	BLOCK_LOGTYPE, "GATEKEEPER: " __VA_ARGS__)
+
 #define G_LOG(level, ...)		        \
 	rte_log_ratelimit(RTE_LOG_ ## level,	\
 	gatekeeper_logtype, "GATEKEEPER: " __VA_ARGS__)
```

Comment:
We are missing code to set `BLOCK_LOGTYPE` to `DEBUG` level.

The macro `G_LOG()` is a better name for our single log macro. Could you add a patch to this pull request to change `G_LOG()` to the new format, and then find `B_LOG()` and replace it with `G_LOG()` in this patch? Notice that the definition of `BLOCK_LOGTYPE` and setting it to `DEBUG` level should be done in the new patch.

Reply:
To clarify, the definition of `BLOCK_LOGTYPE` and setting it to `DEBUG` level should be done in the new patch. Then, how would this patch with `B_LOG()` use the `BLOCK_LOGTYPE`?

Reply:
Yes, the definition of `BLOCK_LOGTYPE` and setting it to `DEBUG` level should be done in the new patch. The new patch must come before the current patch with `B_LOG()`; use `git rebase -i master` to rearrange the patches.

Reply:
Notice that once the new patch comes first, we no longer need `B_LOG()`.